src/test/regress/expected/transactions.out

```diff
@@ -667,3 +667,12 @@ ERROR:  division by zero
 rollback to X;
 ERROR:  no such savepoint
 -- DO NOT ADD ANYTHING HERE.
```

Comment:
Do we really need to add something after the line `DO NOT ADD ANYTHING HERE`?
More, you are trying to change a file which came to us from Postgres and which was not changed by GPDB. The general rule is to avoid changing of Postgres test files if possible.